fnet/network.go

```diff
@@ -217,7 +217,7 @@ func NormalizeHostPort(inputPort string, addr *net.TCPAddr) string {
 // ValidatePayloadSize compares input size with MaxPayLoadSize. If size exceeds the MaxPayloadSize
 // size will set to MaxPayLoadSize
 func ValidatePayloadSize(size *int) {
-	if *size > MaxPayloadSize {
+	if *size > MaxPayloadSize || *size < 0 {
```

Comment:
probably should be a different if and basically use 0 for <0 not max

Reply:
In the `TestGenerateSize` test, 0 is expected to be 0. If I convert `*size<=0', it will cause to fail. Also above comment, you told me to use s >= 0 Aren't we going to handle negative size and update to `MaxPayloadSize`?

Reply:
negative payload should be no payload - not the max payload

Reply:
and the test is wrong, -1 should yield 0 not max